src/main/asciidoc/content.adoc

```diff
@@ -8,9 +8,9 @@ Skip the boring parts and check this out:
 ** <<Graph-Model,Graph>> (compatible with Gremlin and OrientDB SQL)
 ** <<Document-Model,Document>> (compatible with the MongoDB driver and MongoDB queries)
 ** <<KeyValue-Model,Key/Value>> (compatible with the Redis driver)
-** <<SearchEngine-Model,Search Engine>>
+** <<SearchEngine-Model,Search Engine>> (based on the Lucene full-text index)
```

Comment:
My bad for explaining this: we're not using the Lucene engine, but our LSM-Tree-based index. We do use some Lucene libraries for tokenization.